### PR TITLE
a new point in the release_checklist.md

### PR DESCRIPTION
**Pull request by medulka**
https://github.com/nf-core/nf-co.re/issues/533

### CODE REVIEW

File: markdown/developers/release_checklist.md
Comment (Zethson) [inline markdown/developers/release_checklist.md:13]:
```suggestion
* Finalize the description of the pipeline in the GitHub repository and ensure that you remove any "under development" labels in the description. This wording will be used when a new pipeline gets announced on Twitter. 
```

Maybe this is a little bit more direct?

**Review by Zethson — COMMENTED** [review]
Thanks!
Made a couple of suggestions. What do you think?

**Comment by Zethson** [comment]
Cool.

Now you need to fix two style issues:

```
markdown/developers/release_checklist.md:13:219 MD009/no-trailing-spaces Trailing spaces [Expected: 0 or 2; Actual: 1]
markdown/usage/introduction.md:48:1 MD009/no-trailing-spaces Trailing spaces [Expected: 0 or 2; Actual: 4]
```

I don't think that the one in the introduction.md was introduced by you, but it should still be fixed.

**Comment by medulka** [comment]
I committed the changes. The GitHub is still asking for a review.  ???

**Comment by Zethson** [comment]
> I committed the changes. The GitHub is still asking for a review. ???

Yeah, that's fine. I will approve it after you've fixed the style issues.

**Review by medulka — COMMENTED** [review]
The error message was caused by a missing whitespace character at the end of the previous paragraph.

**Comment by Zethson** [comment]
> The error message was caused by a missing whitespace character at the end of the previous paragraph.

Great. There's also one here:

`markdown/usage/introduction.md:48:1 MD009/no-trailing-spaces Trailing spaces [Expected: 0 or 2; Actual: 4]`

**Comment by medulka** [comment]
All checks passed now. I had to delete four trailing spaces.

**Comment by medulka** [comment]
Lukas, I think, you have to review it again, am I right?

**Review by Zethson — APPROVED** [review]
LGTM

Since I usually don't merge PRs here I would CC @drpatelh for a quick approval?

**Pull request merged**
